### PR TITLE
enable CORS

### PR DESCRIPTION
**Pull request by jakkab**
**Description**

Changes proposed in this pull request:
- add CORS support

**Related issue(s)**
#78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,13 +43,16 @@ This procedure is useful to test your own Controller build end-to-end in a local
...
 
-| Name | Required | Description | Possible values |
+| Name | Required | Description | Example values |
 |------|----------|-------------|-----------------|
 | **oathkeeper-svc-address** | yes | ory oathkeeper-proxy service address. | `ory-oathkeeper-proxy.kyma-system.svc.cluster.local` |
 | **oathkeeper-svc-port** | yes | ory oathkeeper-proxy service port. | `4455` |
 | **jwks-uri** | yes | default jwksUri in the Policy. | any string |
 | **service-blacklist** | no | list of services to be blacklisted | `kubernetes` <br> `kube-dns` |
 | **domain-whitelist** | yes | list of domains that can be exposed | `kyma.local` <br> `foo.bar` |
+| **cors-allow-origin**  | no | list of allowed origins | `*`, `https://developer.org` |
```
Comment (Tomasz-Smelcerz-SAP) [inline README.md]:
```suggestion
| **cors-allow-origin**  | no | comma-separated list of allowed origins | `*`, `https://developer.org` |
```

File: main.go [hunk 2]
```diff
@@ -111,6 +116,11 @@ func main() {
 			ServiceBlackList: getList(blackListedServices),
 			DomainWhiteList:  getList(whiteListedDomains),
 		},
+		CorsPolicy: &networkingv1alpha3.CorsPolicy{
```
Comment (Tomasz-Smelcerz-SAP) [inline main.go]: It's better to not tie our config to isito-specific types. Please introduce a "CorsConfig" object, and only create networkingv1alpha3.CorsPolicy object at the time it's really required (during VS creation).

**Comment by Tomasz-Smelcerz-SAP** [comment]
Please extend `make deploy` targets with support for CORS

**Comment by Tomasz-Smelcerz-SAP** [comment]
helm install does not work because of `*` characters in values.yaml

**Comment by jakkab** [comment]
@Tomasz-Smelcerz-SAP Deployment in config/default needs no additional command line flags, CORS flags default to
```
"cors-allow-origin": "*"
"cors-allow-methods": "GET,POST,PUT,DELETE"
"cors-allow-headers": "Authorization,Content-Type,*"
``` if not provided.

**Comment by jakkab** [comment]
`make deploy` to be extended in a separate PR

**Pull request merged**